public/pages/Main/Main.tsx

```diff
@@ -44,6 +46,15 @@ enum Navigation {
   EmailGroups = 'Email recipient groups',
 }
 
+const LocalCluster: DataSourceOption = {
+  label: i18n.translate("dataSource.localCluster", {
+    defaultMessage: "Local cluster",
+  }),
+  id: "",
+};
+
+export const dataSourceObservable = new BehaviorSubject<DataSourceOption>(LocalCluster);
+
```

Comment:
does this by intention to be included in release notes PR?

Reply:
not intended to be in this PR, it happened while resolving merge conflict. I have closed this PR for now